File: Java/03. Code Related to Arrays/69. Check array is disjoint or not.java
import java.util.*;

class Main {
    public static void main(String [] args) {
        Scanner userInput = new Scanner(System.in);

        System.out.print("Enter the number of elements of the Array 1: ");
        int size1 = userInput.nextInt();
        int[] numbers1 = new int[size1];

        for (int i = 0; i < size1; i++) {
            System.out.print("Element " + (i + 1) + ": ");
            numbers1[i] = userInput.nextInt();
        }
        System.out.print("Enter the number of elements of the Array 2: ");
        int size2 = userInput.nextInt();
        int[] numbers2 = new int[size2];

        for (int j = 0; j < size2; j++) {
            System.out.print("Element " + (j + 1) + ": ");
            numbers2[j] = userInput.nextInt();
        }
        int flag = 1;
        for(int i=0; i<size1; i++){
            for(int j=0; j<size2; j++){
                if(numbers1[i] == numbers2[j]){
                    flag = 0;
                    break;
                }
            }
            if (flag == 0) {
                break;
            }
        }
        if (flag == 1){
            System.out.println("The arrays are Disjoint");
        } else {
            System.out.println("The array is not Disjoint");
        }
        userInput.close();
    }
}

File: Python/03. Code Related to Arrays/69. Check array is disjoint or not.py
size1 = int(input("Enter the length of an Array 1:"))
array1 = []
for i in range(size1):
    element = int(input(f"Enter {i+1} element :"))
    array1.append(element)

size2 = int(input("Enter the length of an Array 2:"))
array2 = []
for j in range(size2):
    element = int(input(f"Enter {j+1} element :"))
    array2.append(element)

disjoint = set(array1).isdisjoint(set(array2))

if ( disjoint ):
    print("The entered Arrays are Disjoint")
else:
    print("The entered Arrays are not disjoint")

# Method 2 

# for i in range(0,size1):
#         for j in range(0,size2):
#             if( array1[i] == array2[j] ):
#                 flag = False
#         flag = True
# if (flag):
#     print("The Arrays are disjoint")
# else:
#     print("The arrays are not disjoint")